tests/test_vqe_vqemcscf.py

```diff
@@ -41,6 +41,34 @@ def test_vqecasci_h2o_2e_2o():
     assert utils.almost_equal(mc.e_tot, refmc.e_tot)
 
 
+def test_ssvqecasci_h2o_2e_2o():
+    mc = vqemcscf.VQECASCI(
+        mf,
+        2,
+        2,
+        optimizer=LBFGS(),
+        ansatz=Ansatz.GateFabric,
+        layers=2,
+        excitation_number=3,
+    )
+    mc.kernel()
+
+    # refmc = mcscf.CASCI(mf, 4, 4)
```

Comment:
> The test was indeed not set up correctly. It only produced the ground state energy.

Thanks for noticing.
So was the corresponding CASCI setting of (4,4) also incorrect?
It is very minor, but can you fix them?
```
# refmc = mcscf.CASCI(mf, 4, 4)
# refmc.fcisolver.nroots = 5
```